README.md

```diff
@@ -12,8 +12,8 @@ You need to have Ruby in you operating system to run the irb terminal.
 
 * Else, fork a copy to your repository and clone it locally:
 > `$ git clone https://github.com/TheLuisAbarca/Ruby_school-library/
-$ cd Ruby_school-library`
-$ bundle install
+$ cd Ruby_school-library
+$ bundle install`
 
```

Comment:
- [OPTIONAL] Please it will be nice to update your `readme file` with steps to run your tests as this will be of use to non technical users.

Reply:
Thanks for the suggestion